test/ng/locationSpec.js

```diff
@@ -1601,6 +1601,25 @@ describe('$location', function() {
     });
 
 
+    it('should rewrite links when the specified rewriteLinks attr is detected', function() {
```

Comment:
We need another test (or entend this test if possible), that verifies that the URL is not rewritten if the link doesn't have the correct attribute. (I.e. I think this test would pass even before the PR.)